src/main/java/org/fcrepo/spec/testsuite/versioning/AbstractVersioningTest.java

```diff
@@ -61,12 +61,19 @@ protected Response createVersionedResource(final String uri, final TestInfo info
     }
 
     protected void confirmPresenceOfHeaderValueInMultiValueHeader(final String headerName, final String headerValue,
-                                                                  final Response response) {
+                                                                       final Response response) {
         Assert
             .assertTrue(hasHeaderValueInMultiValueHeader(headerName, headerValue, response),
                         headerName + " with a value of " + headerValue + " must be present but is not!");
     }
 
+    protected void confirmAbsenceOfHeaderValueInMultiValueHeader(final String headerName, final String headerValue,
+                                                                  final Response response) {
```

Comment:
picky, I know, but there's an extra space on this line before 'final'  :)

Reply:
picky is good.

Reply:
picky is good.